auth/src/android/auth_android.cc

```diff
@@ -185,6 +188,31 @@ void UpdateCurrentUser(AuthData* auth_data) {
   }
 }
 
+const char* const kEmulatorLocalHost = "10.0.2.2";
```

Comment:
Just making sure, this is hard-coded for Android on purpose?

Reply:
yes, I find some reference [here](https://cloud.google.com/firestore/docs/emulator#android_apple_platforms_and_web_sdks)